packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -3202,14 +3282,58 @@ void main() {
       await tester.pump();
 
       // Selected the "word" where the tap happened, which is the first space.
+      // Because the "word" is a whitespace, the selection will shift to the
+      // previous "word" that is not a whitespace.
       expect(
         controller.selection,
-        const TextSelection(baseOffset: 7, extentOffset: 8),
```

Comment:
This is suprising. I thought we already select previous word for ios before this change? I don't even know why this test passes before

Reply:
The PR from before only affected editable text, so this test still passed.